packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -189,86 +189,80 @@ const InternalDashboard: React.FC<InternalDashboardProperties> = ({ onSave, edit
-                <Widgets {...widgetsProps} />
-              </ReadOnlyGrid>
-              <WebglContext viewFrame={viewFrame} />
-            </div>
-          </div>
-        </TrendCursorSync>
-      </TimeSync>
-    );
-  }
-
-  return (
-    <TimeSync group='default-dashboard-view'>
       <TrendCursorSync>
-        <div className='dashboard' style={userSelect}>
-          <CustomDragLayer
-            onDrag={(isDragging) => setUserSelect(isDragging ? disabledUserSelect : defaultUserSelect)}
-          />
-          <div className='dashboard-toolbar'>
-            <Box float='left' padding='xs'>
-              <ComponentPalette />
-            </Box>
-            <Box float='right' padding='xs'>
+        <div className='dashboard'>
+          <div className='dashboard-toolbar-read-only'>
+            <Box float='right' padding='s'>
               <SpaceBetween size='s' direction='horizontal'>
                 <ViewportSelection key='1' messageOverrides={DefaultDashboardMessages} />
-                <Divider key='2' />
-                <Actions
-                  key='3'
-                  readOnly={readOnly}
-                  onSave={onSave}
-                  dashboardConfiguration={dashboardConfiguration}
-                  grid={grid}
-                  significantDigits={significantDigits}
-                  editable={editable}
-                />
+                {editable && (
+                  <>
+                    <Divider key='2' />
+                    <Actions
+                      key='3'
+                      readOnly={readOnly}
+                      onSave={onSave}
+                      dashboardConfiguration={dashboardConfiguration}
+                      grid={grid}
+                      significantDigits={significantDigits}
+                      editable={editable}
+                    />
+                  </>
+                )}
               </SpaceBetween>
             </Box>
           </div>
-          <ResizablePanes
-            leftPane={<ResourceExplorer />}
-            centerPane={
-              <div className='display-area' ref={(el) => setViewFrameElement(el || undefined)}>
-                <GestureableGrid {...gridProps}>
-                  <ContextMenu {...contextMenuProps} />
-                  <Widgets {...widgetsProps} />
-                  {activeGesture === 'select' && <UserSelection {...selectionProps} />}
-                </GestureableGrid>
-                <WebglContext viewFrame={viewFrame} />
-              </div>
-            }
-            rightPane={propertiesPanel}
-          />
+          <div className='display-area' ref={(el) => setViewFrameElement(el || undefined)}>
+            <ReadOnlyGrid {...grid}>
+              <Widgets {...widgetsProps} />
+            </ReadOnlyGrid>
+            <WebglContext viewFrame={viewFrame} />
+          </div>
         </div>
       </TrendCursorSync>
-    </TimeSync>
+    );
+  }
+
+  return (
+    <TrendCursorSync>
```

Comment:
Is there any timesync tests you think we could add to make sure changing to one timesync works correctly?